app/models/webhook.rb

```diff
@@ -0,0 +1,5 @@
+class Webhook
```

Comment:
This should be `class Webhook < ActiveRecord::Base; ...; end` instead of using a pseudo-model YAML config.